book-domain-service/app/routes.py

```diff
@@ -5,10 +5,10 @@
 @app.route('/<int:id>/checkout', methods=['PUT'])
 def checkout(id):
 
-    bookTitle = Book.query.get(id).title
+    book = Book.query.get(id)
 
-    if(bookTitle):
-        return Response("You checked out book id: " + str(id) + " name: " +
-        str(bookTitle), 200)
+    if(book):
+        bookTitle = book.title
```

Comment:
Python typically uses underscores for variables rather than camel case. So you'd typically see this `book_title` instead of `bookTitle`